File: spring-cloud-square-demo/consumer-okhttp/src/main/java/com/livk/consumer/OkhttpConsumerApp.java
package com.livk.consumer;

import com.livk.common.LivkSpring;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * <p>
 * OkhttpConsumerApp
 * </p>
 *
 * @author livk
 * @date 2022/4/13
 */
@SpringBootApplication
public class OkhttpConsumerApp {
    public static void main(String[] args) {
        LivkSpring.run(OkhttpConsumerApp.class, args);
    }
}

File: spring-cloud-square/consumer-okhttp/src/main/kotlin/com/livk/consumer/OkhttpConsumerApp.kt
package com.livk.consumer

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/**
 * @author livk
 */
@SpringBootApplication
open class OkhttpConsumerApp

fun main(args: Array<String>) {
    runApplication<OkhttpConsumerApp>(*args)
}
